src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/Microsoft.VisualStudio.Razor.IntegrationTests.csproj

```diff
@@ -39,4 +36,35 @@
   <ItemGroup>
     <EmbeddedResource Include="**\TestFiles\**\*" />
   </ItemGroup>
+
+  <ItemGroup>
+    <!-- Microsoft.VisualStudio.RazorExtension.vsix cannot be installed without elevation due to a bug in the experimental extension installation logic. -->
+    <!--<AssemblyAttribute Include="Xunit.Harness.RequireExtensionAttribute">
+      <_Parameter1>Microsoft.VisualStudio.RazorExtension.vsix</_Parameter1>
+    </AssemblyAttribute>-->
+    <AssemblyAttribute Include="Xunit.Harness.RequireExtensionAttribute">
+      <_Parameter1>Microsoft.VisualStudio.RazorExtension.Dependencies.vsix</_Parameter1>
+    </AssemblyAttribute>
+  </ItemGroup>
+
+  <Target Name="PrepareVsixProjectReferences"
+          BeforeTargets="ResolveProjectReferences"
+          DependsOnTargets="PrepareProjectReferences">
+    <MSBuild
+      Projects="@(_MSBuildProjectReferenceExistent)"
+      Targets="VSIXContainerProjectOutputGroup"
+      BuildInParallel="$(BuildInParallel)"
+      Properties="%(_MSBuildProjectReferenceExistent.SetConfiguration); %(_MSBuildProjectReferenceExistent.SetPlatform); %(_MSBuildProjectReferenceExistent.SetTargetFramework); CreateVsixContainer=true"
+      Condition="'%(_MSBuildProjectReferenceExistent.CopyVsix)' == 'true'"
+      ContinueOnError="!$(BuildingProject)"
+      RemoveProperties="%(_MSBuildProjectReferenceExistent.GlobalPropertiesToRemove)">
+
+      <Output TaskParameter="TargetOutputs" ItemName="_ProjectReferenceVsixOutputs" />
+    </MSBuild>
+
+    <ItemGroup>
+      <ReferenceCopyLocalPaths Include="@(_ProjectReferenceVsixOutputs)" />
+    </ItemGroup>
+  </Target>
```

Comment:
📝 I need to verify this is exactly what we want as a general solution, and get this moved over to microsoft/vs-extension-testing.